srv/www/regionService/regionInfo.py

```diff
@@ -124,6 +124,13 @@ def create_smt_region_map(conf):
                 'Missing smt-server-name data in section %s.' % section
             )
             sys.exit(1)
+        try:
+            region_smt_registry_names = region_data_cfg.get(section, 'smt-registry-name')
+        except Exception:
+            logging.error(
+                'Missing smt-registry-name data in section %s.' % section
+            )
+            sys.exit(1)
```

Comment:
We do not want to make the registry a mandatory component of the region server setup, no exit. Also the message should be `info` not `error`